checkov/terraform/checks/resource/gcp/GoogleProjectAdminServiceAccount.py

```diff
@@ -0,0 +1,24 @@
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+from checkov.common.models.enums import CheckResult, CheckCategories
+import re
+
+USER_MANAGED_SERVICE_ACCOUNT = re.compile ('.*@\d+\.iam\.gserviceaccount\.com$')
```

Comment:
@arielkru This is a user managed service account:
`test2-218@gcp-bridgecrew-deployment.iam.gserviceaccount.com`
It doesn't fit this regex. Am I wrong?

Reply:
according to gcp docs there's only one format for user managed service accounts:
https://cloud.google.com/iam/docs/service-accounts#user-managed

I'm not sure how it aligns with your service account. WDYT?